test/use-swr-int.test.tsx

```diff
@@ -0,0 +1,354 @@
+import { act, render, screen } from '@testing-library/react'
```

Comment:
What does the `int` in the file name (`use-swr-int.test.tsx`) mean?

Reply:
curious as well 😄 @promer94 do you mean `initial` or `basic`?

Reply:
I keep the olds tests because i think #881 and #896 might need them as references.

The name comes from #581

Reply:
@promer94 
Thank you!
Does the `int` means `integration`?
IMO, I prefer more explicit name like `use-swr-integration.test.tsx` because some people including me might misunderstand the meaning of the name. It's not a problem  If the name is common for an integration test.

Reply:
Yeah I agree `use-swr-integration.test.tsx` would be better!